core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -493,6 +495,51 @@ export class BaseUser {
   getCurrentUrlWithoutParameters(): string {
     return this.page.url().split('?')[0];
   }
+
+  /**
+   * This function checks the exploration accessibility by navigating to the
+   * exploration page based on the explorationID.
+   */
+  async expectExplorationToBeAccessibleByUrl(
```

Comment:
I just realized that this function and the one below(expectExplorationToBeNotAccessibleByUrl) are in puppeteer-utils.ts. Why aren't they in exploration-editor.ts? You only called these two functions with explorationEditor user so I think these should be in the file exploration-editor.ts.

Reply:
Initially, these methods were located in exploration-editor.ts. However, after discussions with @seanlip, it was decided to relocate them to puppeteer.utils.ts as they can be utilized by anyone, not limited to exploration-editor.